app/models/report_generators/lsa/fy2021/all.rb

```diff
@@ -164,6 +164,7 @@ def create_hmis_csv_export
         start_date: '2012-10-01', # using 10/1/2012 so we can determine continuous homelessness
         end_date: @report_end,
         projects: @project_ids,
+        coc_codes: @coc_code,
```

Comment:
is `@coc_code` sometimes an array?

Reply:
Not in the LSA as it stands, but I tried to make sure the code would tolerate both.